### PR TITLE
Cancelling non-staked expenditure as non-owner (using motions)

### PR DESCRIPTION
**Pull request by jakubcolony**
https://github.com/JoinColony/colonyCDapp/pull/2020

### CODE REVIEW

File: src/graphql/generated.ts [hunk 1]
```diff
@@ -373,7 +373,7 @@ export enum ColonyActionType {
   CreateDomain = 'CREATE_DOMAIN',
   /** An action related to creating a domain within a Colony via a motion */
   CreateDomainMotion = 'CREATE_DOMAIN_MOTION',
-  /** An action related to creating expenditure (dvanced payment)  */
+  /** An action related to creating expenditure (dvanced payment) */
```
Comment (iamsamgibbs) [inline src/graphql/generated.ts:376]: Tiny typo

**Review by iamsamgibbs — APPROVED** [review]
Tested with colonyCDapp#2020

File: src/graphql/generated.ts (hunk 1 above)
Comment (jakubcolony) [reply src/graphql/generated.ts:376]: Good spot, think I fixed it in the schema so just need to re-run codegen 👍

**Review by rdig — APPROVED** [review]
All good on this side as well. Nice work!

**Pull request merged**